source/apiVolontaria/apiVolontaria/services.py

```diff
@@ -1,35 +1,27 @@
 from django.conf import settings
-from django.core.mail import EmailMessage
+from django.core.mail import send_mail
 from django.utils.translation import ugettext_lazy as _
-
 from .exceptions import MailServiceError
+from django.template.loader import render_to_string
 
-
-def send_mail(users, context, template):
+def service_send_mail(emails, subject, path_templates, merge_data):
```

Comment:
Why not use "template_name" instead of "path_template"? Templates will always have a *.html and a *.txt version.

Reply:
parce que dans la théorie cette liste doit contenir un path (même si à la base une partie du path est fournit dynamiquement). Dans la mesure où une personne aurait des sous repertoires, il devra mettre dans son path (dans la liste path_templates) le nom du sous dossier en question.

Reply:
Ok, je suis bien d'accord avec ça!